setup.py

```diff
@@ -1,25 +1,45 @@
+# encoding: utf-8
+
 from setuptools import setup, find_packages
+from codecs import open  # To use a consistent encoding
+from os import path
 
 version = '0.2.0'
 
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the relevant file
+with open(path.join(here, 'README.markdown'), encoding='utf-8') as f:
+    long_description = f.read()
+
 setup(
     name='ckanext-datapackager',
     version=version,
-    description="",
-    long_description='''
-    ''',
-    classifiers=[],  # Get strings from http://pypi.python.org/pypi?%3Aaction=list_classifiers
-    keywords='',
-    author='',
-    author_email='',
-    url='',
-    license='',
+    description="This extension adds importing and exporting of Data Packages to CKAN datasets.",
+    long_description=long_description,
+
+    # see http://pypi.python.org/pypi?%3Aaction=list_classifiers
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
```

Comment:
I'm not sure I would call this stable yet. Maybe we should wait until this is deployed to a big CKAN instance (like the DataHub)? After that, if nothing appears, I'm happy to change to stable and launch a version 1.x